src/main/resources/db/dbchangelog-dev.yaml

```diff
@@ -4,4 +4,6 @@ databaseChangeLog:
   - include:
       file: db/changelog/002-insert-into-wallets.yaml
   - include:
-      file: db/changelog/003-create-table-financial_transactions.yaml
\ No newline at end of file
+      file: db/changelog/003-create-table-financial_transactions.yaml
+  - include:
+      file: db/changelog/004-insert-into-financial_transactions.yaml
```

Comment:
suggestion for filenames:
`003-create-financial_transactions-schema.xml`

`002-insert-wallet-data.yaml`
`004-insert-financial_transaction-data.yaml`

Reply:
Could you please tell me why you want to add ...-schema and ...-data for the file name? I a little bit don't understand and I would like to understand :)